test/unit/spec/function_output/kbaseReportView-spec.js

```diff
@@ -1,8 +1,311 @@
-define(['kbaseReportView'], (Widget) => {
+define(['kbaseReportView', 'base/js/namespace', 'jquery', 'narrativeMocks', 'narrativeConfig'], (
+    ReportView,
+    Jupyter,
+    $,
+    Mocks,
+    Config
+) => {
     'use strict';
+    const FAKE_SERV_URL = 'https://ci.kbase.us/report_serv';
+    const REPORT_REF = '1/2/3';
+
+    const REPORT_OBJ = {
```

Comment:
This is roughly the same report object used in other tests. It's put in here on its own to get heavily modified in between tests.